projects/igniteui-angular/src/lib/services/transaction/README.md

```diff
@@ -47,3 +47,7 @@ and then inject it in the component's constructor:
    |clear                  | Clears all transactions                                       | id?                       |
    |startPending           | Starts pending transactions. All transactions passed after call to startPending will not be added to transaction log | - |
    |endPending             | Clears all pending transactions and aggregated pending state. If commit is set to true commits pending states as single transaction | commit |
+
```

Comment:
you should point to the name of the new interface here. Please add:
```
### HierarchicalTransactionService
```
together with the necessary new lines as done for `TransactionService` above.